Karatsuba_Algo/Karatsuba_multiplication_algo.java

```diff
@@ -0,0 +1,85 @@
+/**
+*    Java Program to Implement Karatsuba Multiplication Algorithm
+**/
+import java.util.*;
+
+public class Karatsuba{
+    /** Function to multiply two numbers **/
+    public long multiply(long x, long y){
+        int size1 = getSize(x);
+        int size2 = getSize(y);
+
+        /** Maximum of lengths of number **/
+
+        int N = Math.max(size1, size2);
+
+        /** for small values directly multiply **/        
+
+        if (N < 10)
+            return x * y;
+        /** max length divided, rounded up **/    
+        N = (N / 2) + (N % 2);    
+        /** multiplier **/
```

Comment:
I feel it's better if you improve this commend. Instead of saying what are you doing, it's better if you say why are you doing this